### ISSUE TITLE
[Suggestion] Different ways to sell items

### ISSUE THREAD

**Issue by Reisai** [The-Fireplace-Minecraft-Mods/Grand-Exchange#6]
So, while using this mod, I found that it was missing a feature or two I'd really like to see. First of all, there is no way (that I know of) to create a shop with infinite money that players can sell items to. This would add another way for players to earn money without getting daily currency and adds more money to the server economy as players obtain more items. Another feature that would be interesting is allowing signs to be linked to an offer you've created. Unlike sign shops in plugins, these wouldn't be linked to a chest and instead could be placed anywhere, by a player, to allow actual storefronts for players to discover new offers on their own.

**Comment by The-Fireplace** [comment]
I fully agree with you that both of these things should be added. The first thing, I partially addressed with the /convert feature of Grand Exchange, where you can convert items into currency, but that doesn't come close to the ease of use a Grand Exchange offer with infinite resources, whether that is infinite money to buy items with or infinite items to sell. As for the second suggestion, I hadn't even thought of that, and it would be a great addition to the mod, allowing users who enjoy browsing sign shops to do so while keeping the convenience of having offers easily accessible from anywhere. I can't promise it anytime soon, but I do intend to add both of these features at some point.

**Comment by The-Fireplace** [comment]
> First of all, there is no way (that I know of) to create a shop with infinite money that players can sell items to. This would add another way for players to earn money without getting daily currency and adds more money to the server economy as players obtain more items

This part is done as of version 1.4.0
